### PR TITLE
MAUI-1750:Update See also Content,TabTitle and highlight codes in UG

### PR DESCRIPTION
**Pull request by RiyashameedM**
Update See also Content,TabTitle and highlight codes in UG

### CODE REVIEW

File: MAUI/ListView/viewappearance.md [hunk 1]
```diff
@@ -750,7 +750,7 @@ The `SfListView` supports accordion view to display a list of items. Each item c
 `Accordion` view can be displayed by defining two different ItemTemplates. The ItemTemplates can be enabled or disabled in the [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event.
 
 {% tabs %}
-{% highlight c# %}
+{% highlight c# tabtitle="SfListViewAccordionBehavior.cs" hl_lines="33 34 35 36 37 38 39 40 41 42 43 44 45 46 47" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:753]: AccordionBehavior.cs

Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:753]: AccordionBehavior.cs

File: MAUI/ListView/viewappearance.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ Assign custom `DataTemplateSelector` to the [ItemTemplate](https://help.syncfusi
   </Grid>
 </ContentPage>
 {% endhighlight %}
-{% highlight c# %}
+{% highlight c# tabtitle= "MainPageCs.cs" hl_lines="11" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:81]: MainPage.xaml.cs

rename the class name to MainPage

Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:81]: MainPage.xaml.cs

rename the class name to MainPage

File: MAUI/ListView/viewappearance.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Here, an [ItemsCacheLimit](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.L
 Create a custom class that inherits from `DataTemplateSelector`, and override the `OnSelectTemplate` method to return the `DataTemplate` for that item. At runtime, the SfListView invokes the `OnSelectTemplate` method for each item and passes the data object as parameter.
 
 {% tabs %}
-{% highlight c# %}
+{% highlight c# tabtitle= "MyDataTemplateSelector" hl_lines="13 14 15 16 17 18 19" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/viewappearance.md:29]: DataTemplateSelector.cs

File: MAUI/ListView/selection.md [hunk 4]
```diff
@@ -369,7 +369,7 @@ The [SelectionChangedCommand](https://help.syncfusion.com/cr/maui/Syncfusion.Mau
 </ContentPage>
 
 {% endhighlight %}
-{% highlight c# %}
+{% highlight c# tabtitle= "CommandViewModel.cs" hl_lines="12 13 14 15" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/selection.md:372]: ViewModel.cs

Comment (PiruthivirajM) [inline MAUI/ListView/selection.md:372]: ViewModel.cs

File: MAUI/ListView/selection.md [hunk 5]
```diff
@@ -53,7 +53,7 @@ listView.SelectedItem = viewModel.Items[5];
 When the selection mode is `Multiple`, programmatically select more than one item by adding the underlying object to the `SfListView.SelectedItems` property. 
 
 {% tabs %}
-{% highlight c# %} 
+{% highlight c# tabtitle= "SelectionViewModel.cs" hl_lines="42 43" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/selection.md:56]: ViewModel.cs

Comment (PiruthivirajM) [inline MAUI/ListView/selection.md:56]: ViewModel.cs

File: MAUI/ListView/mvvm.md [hunk 6]
```diff
@@ -693,7 +687,7 @@ listView.SetBinding(SfListView.IsBusyProperty, new Binding("IsBusy", BindingMode
 {% endtabs %}
 
 {% tabs %}
-{% highlight c# %}
+{% highlight c# tabtitle= "ViewModel" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:690]: ViewModel.cs

Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:690]: ViewModel.cs

File: MAUI/ListView/mvvm.md [hunk 7]
```diff
@@ -198,7 +198,7 @@ You can bind command of `Button` inside `ItemTemplate` to the command in ViewMod
     </Syncfusion:SfListView.ItemTemplate>
 </Syncfusion:SfListView>
 {% endhighlight %}
-{% highlight c# %}
+{% highlight c# tabtitle= "ContactsViewModel" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:201]: ViewModel.cs

Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:201]: ViewModel.cs

File: MAUI/ListView/mvvm.md [hunk 8]
```diff
@@ -154,7 +154,7 @@ You can bind a command for a `Button` inside `ItemTemplate` to the command in Mo
 </listView:SfListView>
 
 {% endhighlight %}
-{% highlight c# %}
+{% highlight c# tabtitle= "ContactInfo_NestedListView" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:157]: ContactsInfo.cs

Comment (PiruthivirajM) [inline MAUI/ListView/mvvm.md:157]: ContactsInfo.cs

File: MAUI/ListView/loadmore.md [hunk 9]
```diff
@@ -493,7 +492,7 @@ private bool CanLoadMoreItems(object obj)
 The `SfListView` allows adding more items infinite times,either manually or automatically. 
 
 {% tabs %}
-{% highlight c# %}
+{% highlight c# tabtitle= "LoadMoreViewModel.cs" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/loadmore.md:495]: ViewModel.cs

Comment (PiruthivirajM) [inline MAUI/ListView/loadmore.md:495]: ViewModel.cs

File: MAUI/ListView/layouts.md [hunk 10]
```diff
@@ -68,7 +68,7 @@ listView.ItemsLayout = new GridLayout() { SpanCount = 2 };
 In the `SfListView`, the [GridLayout](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.GridLayout.html) allows you to change the `SpanCount` based on the view size of application with orientation in either portrait or landscape mode.
 
 {% tabs %}
-{% highlight c# %}
+{% highlight c# tabtitle= "GridLayoutPage.cs" hl_lines="8 9 10" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/layouts.md:71]: MainPage.xaml.cs

Comment (PiruthivirajM) [inline MAUI/ListView/layouts.md:71]: MainPage.xaml.cs

File: MAUI/ListView/grouping.md [hunk 11]
```diff
@@ -1063,7 +1063,7 @@ listView.GroupHeaderTemplate = new DataTemplate(() =>
  });
 {% endhighlight %}
 
-{% highlight c# %}
+{% highlight c# tabtitle= "SelectionBoolToBackgroundColorConverter.cs" %}
```
Comment (PiruthivirajM) [inline MAUI/ListView/grouping.md:1066]: BoolToColorConverter.cs

Comment (PiruthivirajM) [inline MAUI/ListView/grouping.md:1066]: BoolToColorConverter.cs

File: MAUI/ListView/viewappearance.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md:753]: Resolved the change

File: MAUI/ListView/viewappearance.md (hunk 2 above)
Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md:81]: resolved the changes

Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md:81]: resolved the changes

File: MAUI/ListView/viewappearance.md (hunk 3 above)
Comment (RiyashameedM) [reply MAUI/ListView/viewappearance.md:29]: resolved the change

File: MAUI/ListView/selection.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/ListView/selection.md:372]: resolved the change

File: MAUI/ListView/selection.md (hunk 5 above)
Comment (RiyashameedM) [reply MAUI/ListView/selection.md:56]: resolved the change

Comment (RiyashameedM) [reply MAUI/ListView/selection.md:56]: resolved the change

File: MAUI/ListView/mvvm.md (hunk 6 above)
Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:690]: resolved the change

File: MAUI/ListView/mvvm.md (hunk 7 above)
Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:201]: resolved the mistake

Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:201]: resolved the mistake

File: MAUI/ListView/mvvm.md (hunk 8 above)
Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:157]: resolved the change

Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:157]: resolved the change

File: MAUI/ListView/loadmore.md (hunk 9 above)
Comment (RiyashameedM) [reply MAUI/ListView/loadmore.md:495]: resolved the change

Comment (RiyashameedM) [reply MAUI/ListView/loadmore.md:495]: resolved the change

File: MAUI/ListView/layouts.md (hunk 10 above)
Comment (RiyashameedM) [reply MAUI/ListView/layouts.md:71]: resolved the change

Comment (RiyashameedM) [reply MAUI/ListView/layouts.md:71]: resolved

File: MAUI/ListView/grouping.md (hunk 11 above)
Comment (RiyashameedM) [reply MAUI/ListView/grouping.md:1066]: resolved the change

Comment (RiyashameedM) [reply MAUI/ListView/grouping.md:1066]: resolved the change

File: MAUI/ListView/mvvm.md (hunk 6 above)
Comment (RiyashameedM) [reply MAUI/ListView/mvvm.md:690]: resolved

**Review by PiruthivirajM — APPROVED** [review]
changes fine

**Pull request merged**